forte/model.py

```diff
@@ -19,28 +22,42 @@ class Model(ABC):
     def point_group(self) -> str:
         """The model point group"""
 
+    @abstractmethod
+    def ints(self, mo_space_info):
+        """Make a ForteIntegral object"""
+
 
 class MolecularModel(Model):
     """
     A class used to handle molecules.
     """
-    def __init__(self, molecule: Molecule, basis: Basis, scf_aux_basis: Basis = None, corr_aux_basis: Basis = None):
+    def __init__(
+        self,
+        molecule: Molecule,
+        basis: Basis,
+        int_type: str = None,
+        scf_aux_basis: Basis = None,
+        corr_aux_basis: Basis = None
+    ):
         """
         Initialize a MolecularModel object
 
         Parameters
         ----------
         molecule: Molecule
             the molecule information
+        int_type: str
+            the type of integrals (e.g., 'CONVENTIONAL', 'DF', 'CD',...)
```

Comment:
Standard syntax is to include the list of allowed values in place of the argument type, e.g.,
`int_type: {'CONVENTIONAL', 'DF, 'CD'}`

Reply:
Added.